### ISSUE TITLE
Can't generate documentation?

### ISSUE THREAD

**Issue by nrbrook** [ReSwift/ReSwift#243]
Running build.sh in gh-pages branch, I get

```
$ ./build.sh 
Cloning into 'vendor/jazzy'...
warning: Could not find remote branch integrated-markdown to clone.
fatal: Remote branch integrated-markdown not found in upstream origin
.build_helpers: line 14: bundle: command not found
Cloning into 'ReSwift'...
remote: Counting objects: 100, done.
remote: Compressing objects: 100% (97/97), done.
remote: Total 100 (delta 7), reused 53 (delta 0), pack-reused 0
Receiving objects: 100% (100/100), 1.97 MiB | 371.00 KiB/s, done.
Resolving deltas: 100% (7/7), done.
~/Desktop/ReSwift-gh-pages/ReSwift ~/Desktop/ReSwift-gh-pages
.build_helpers: line 32: ./generate_docs.sh: No such file or directory
~/Desktop/ReSwift-gh-pages
```

**Comment by artonragsdale** [comment]
I noticed the documentation at http://reswift.github.io/ReSwift/ is still on version 2.0. Is that no longer being used or is there just a technical problem like above preventing its generation?

**Comment by Ben-G** [comment]
Hey! 

The docs haven't been updated because I missed to do that. In the past Karl had updated these and I had simply forgotten (need release checklist/script). Karl was also the primary maintainer of the script.

But I think I can see what is going on with the script. I'll take a look and report back.

**Comment by Ben-G** [comment]
@nrbrook @artonragsdale I addressed the issue in this PR (#247). Thanks a lot for pointing it out. Unfortunately a failure in doc generation doesn't fail the Travis build. I'll see if there's a good way to report failures in future.

The scripts in the `gh-pages` branch are actually outdated. Instead the `.scripts/update-gh-pages` script from the `master` branch should be used. I'll remove the scripts in a separate step.

**Comment by Ben-G** [comment]
This is now fixed after merging #247 and #249. Thanks for reporting!
